feat(content): Pirate Expansion - Remains of the CCOR (#9597)
endless-sky/endless-sky: data/human/ships.txt

ship "Enforcer"
	sprite "ship/enforcer"
	thumbnail "thumbnail/enforcer"
	attributes
		category "Light Warship"
		"cost" 490000
		"shields" 2300
		"hull" 1100
		"required crew" 3
		"bunks" 5
		"mass" 130
		"drag" 2.9
		"heat dissipation" .7
		"fuel capacity" 400
		"cargo space" 30
		"outfit space" 210
		"weapon capacity" 55
		"engine capacity" 60
		weapon
			"blast radius" 34
			"shield damage" 340
			"hull damage" 170
			"hit force" 510
	outfits
		"Energy Blaster" 4
		"Laser Turret"
		
		"nGVF-CC Fuel Cell"
		"LP036a Battery Pack"
		"D14-RN Shield Generator"
		
		"X2700 Ion Thruster"
		"X2200 Ion Steering"
		"Hyperdrive"

	engine 0 39
	gun -11 -52
	gun 11 -52
	gun -5.5 -48
	gun 5.5 -48
	turret 0 -0.5
	leak "leak" 40 30
	leak "flame" 50 60
	explode "tiny explosion" 8
	explode "small explosion" 12
	explode "medium explosion" 4

> 	description "Prior to the formation of the galactic Republic, many star sectors were owned by small governing corporate entities, with the vast majority of space being inaccessible to private citizens. This led to an increase in pirate activity, and the Enforcer was one of the more common system security vessels used for defense. The ship boasts an armament of four gun ports and a turreted mount, all packed on a small but defensible frame that was powerful for its time. Although no longer on the market, the Enforcer finds itself in the possession of a small number of merchants, or, ironically, various groups of pirates, as an aging combat ship easily outperformed by other light warships or even interceptors of a similar size."

Reviewer: > This led to an increase in pirate activity

How? The link between space travel being prohibitively expensive and increased piracy isn't obvious to me.

Author: If not Pirate activity, do you have any suggestions to why they may be designated as system security vessels?  Maybe something to do with anxiety or just early prevention? 

If not, maybe we change up the lore purpose?

Reviewer 2: Something like this?
```suggestion
	description "Prior to the formation of the galactic Republic, many star sectors were owned by small governing corporate entities, with the vast majority of space being inaccessible to private citizens. The only exception to this rule were the pirate bands, whose few ships were often stolen or rebellious corporate fleets of their own, yet who had an outsized impact on interstellar trade. The necessity for corporations to protect their operations meant that Enforcer was one of the more common system security vessels used for defense. The ship boasts an armament of four gun ports and a turreted mount, all packed on a small but defensible frame that was powerful for its time.`
	description `	Although no longer on the market, the Enforcer finds itself in the possession of a small number of merchants, or, ironically, various groups of pirates, as an aging combat ship easily outperformed by other light warships or even interceptors of a similar size."
```

Author: Gonna commit this but gonna check it in-game to see if it still fits the box.

Author: It's a bit long, but I think it's fine for a lore grab. It's not that much longer than the Saber's.

ship "Enforcer"
	sprite "ship/enforcer"
	thumbnail "thumbnail/enforcer"
	attributes
		category "Light Warship"
		"cost" 490000
		"shields" 2300
		"hull" 1100
		"required crew" 3
		"bunks" 5
		"mass" 130
		"drag" 2.9
		"heat dissipation" .7
		"fuel capacity" 400
		"cargo space" 30
		"outfit space" 210
		"weapon capacity" 55
		"engine capacity" 60
		weapon
			"blast radius" 34
			"shield damage" 340
			"hull damage" 170
			"hit force" 510
	outfits
		"Energy Blaster" 4
		"Laser Turret"
		
		"nGVF-CC Fuel Cell"
		"LP036a Battery Pack"
		"D14-RN Shield Generator"
		
		"X2700 Ion Thruster"
		"X2200 Ion Steering"
		"Hyperdrive"

	engine 0 39
	gun -11 -52
	gun 11 -52
	gun -5.5 -48
	gun 5.5 -48
	turret 0 -0.5
	leak "leak" 40 30
	leak "flame" 50 60
	explode "tiny explosion" 8
	explode "small explosion" 12
	explode "medium explosion" 4

> 	description "Prior to the formation of the galactic Republic, many star sectors were owned by small governing corporate entities, with the vast majority of space being inaccessible to private citizens. The only exception to this rule were the pirate bands, whose few ships were often stolen or rebellious corporate fleets of their own, yet who had an outsized impact on interstellar trade. The necessity for corporations to protect their operations meant that Enforcer was one of the more common system security vessels used for defense. The ship boasts an armament of four gun ports and a turreted mount, all packed on a small but defensible frame that was powerful for its time."

Reviewer 2: ```suggestion
	description `Prior to the formation of the Republic, many star systems were owned by "governing corporate entities," with the vast majority of space being inaccessible to private citizens. The only exception to this rule were the pirate bands, whose few ships were often stolen or rebellious corporate fleets of their own, yet who had an outsized impact on interstellar trade. The necessity for corporations to protect their operations meant that Enforcer was one of the most common vessels used for security and defense. The ship boasts an armament of four gun ports and a turreted mount, all packed on a small but defensible frame that was powerful for its time.`
```
Matching how the Timeline presents the "governing corporate entities." I don't see why they should be "small", though.

I think "galactic Republic" is only used in one hostile Navy hail, I was thinking of removing that mention actually. The Republic doesn't cover the whole galaxy, we don't call any governments on Earth planetary governments since they don't cover the planet.

Reviewer: The phrase "galactic Republic" is used once in the historical timeline on the wiki as well. So there is precedence for it, but it's extremely rare, and the "galactic" part is always lowercased, implying it's a description rather than an official title. 

I think it actually would be fair to keep the phrase "galactic Republic" in this case. Sure, it doesn't encompass the entire galaxy, but it does encompass almost the entirety of human space, with the sparse few pirate worlds being the only planets outside of the Republic at game start, and I wouldn't be surprised if the Republic claims jurisdiction over those pirate systems, meaning that by their own description they would be a galactic government.

Reviewer 2: Sure.

~~(The actual reason was I was trying to cut the description down.)~~

> 	description "	Although no longer on the market, the Enforcer finds itself in the possession of a small number of merchants, or, ironically, various groups of pirates, as an aging combat ship easily outperformed by other light warships or even interceptors of a similar size."

Reviewer 2: I am reviewing one more time so I can approve one more time.
```suggestion
	description `	Although no longer on the market, the Enforcer finds itself in the possession of a small number of merchants and, ironically, various groups of pirates, as an aging combat ship easily outperformed by other light warships or even interceptors of a similar size.`
```
Is the description is saying the Enforcer is only a merchant ship or only a pirate ship but not both? Seems conflicting.

Comma looks funny if you remove "ironically": `Although no longer on the market, the Enforcer finds itself in the possession of a small number of merchants, or various groups of pirates`

ship "Hogshead"
	sprite "ship/hogshead"
	thumbnail "thumbnail/hogshead"
	attributes
		category "Space Liner"
		"cost" 3150000
		"shields" 3200
		"hull" 1900
		"required crew" 18
		"bunks" 84
		"mass" 210
		"drag" 4.3
		"heat dissipation" .55
		"fuel capacity" 600
		"cargo space" 45
		"outfit space" 320
		"weapon capacity" 80
		"engine capacity" 90
		weapon
			"blast radius" 51
			"shield damage" 510
			"hull damage" 255
			"hit force" 765
	outfits
		"Heavy Anti-Missile Turret"
		"Anti-Missile Turret"

		"S3 Thermionic"
		"KP-6 Photovoltaic Array" 2
		"LP036a Battery Pack"

		"D41-HY Shield Generator"
		"Small Radar Jammer" 2
		"Luxury Accommodations"
		
		"Greyhound Plasma Thruster"
		"Greyhound Plasma Steering"
		"Hyperdrive"

	engine 34 42
	engine -34 42
	gun -17.5 -79.5
	gun 17.5 -79.5
	turret 0 -44 "Heavy Anti-Missile Turret"
	turret 0 63 "Anti-Missile Turret"
	leak "leak" 50 40
	leak "big leak" 70 40
	explode "tiny explosion" 10
	explode "small explosion" 25
	explode "medium explosion" 25
	explode "large explosion" 10
	"final explode" "final explosion medium"

> 	description `Manufactured from a forgotten shipyard many centuries ago, the Hogshead got its iconic name from the keg-like shape of its engine pods being a hot topic of discussion for captains everywhere. The ship quickly became popular after first being released for consumer purchase and was commonly seen around various systems as the go-to "space bus" in human space.`

Reviewer 2: The physical shipyard might still exist, so if you amend your suggestion to say `First manufactured by a forgotten shipyard many centuries ago`, it would be better.

> 	description `	Unfortunately, only a few decades after introducing the ship, the shipyard went bankrupt and was never seen again, leaving the Hogshead to history as their one-hit-wonder. In the current era, Hogsheads are much more uncommon, typically treated as "unsightly" compared to other vessels such as the Star Queen, but a few places in human space still sell them.`

Author: Edited in "though" so it still flows properly.

> 	description `	Unfortunately, only a few decades after introducing the ship, the shipyard went bankrupt and was never heard from again, leaving the Hogshead to history as their one-hit-wonder. In the current era, Hogsheads are much less common; considered "unsightly" compared to modern vessels such as the Star Queen, though a few places in human space still sell them.`

Reviewer 3: ```suggestion
	description `	Unfortunately, only a few decades after introducing the ship, the shipyard went bankrupt and was never heard from again, leaving the Hogshead to history as their one-hit-wonder. In the current era, Hogsheads are much less common; they are considered "unsightly" compared to modern vessels such as the Star Queen, though a few places in human space still sell them.`
```

ship "Saber"
	sprite "ship/saber"
	thumbnail "thumbnail/saber"
	attributes
		category "Medium Warship"
		"cost" 2200000
		"shields" 4600
		"hull" 2600
		"required crew" 16
		"bunks" 25
		"mass" 370
		"drag" 6
		"heat dissipation" .5
		"fuel capacity" 600
		"cargo space" 30
		"outfit space" 280
		"weapon capacity" 105
		"engine capacity" 90
		weapon
			"blast radius" 72
			"shield damage" 720
			"hull damage" 360
			"hit force" 1080
	outfits
		"Beam Laser" 3
		"Javelin Pod" 2
		"Javelin" 600
		"Javelin Storage Crate" 2
		"Blaster Turret"
		
		"nGVF-CC Fuel Cell"
		"LP072a Battery Pack"
		"D23-QP Shield Generator"
		
		"X3700 Ion Thruster"
		"X3200 Ion Steering"
		"Hyperdrive"

	engine 0 105
	gun -10.5 -105 "Beam Laser"
	gun 0 -105 "Beam Laser"
	gun 10.5 -105 "Beam Laser"
	gun -5 -104 "Javelin Pod"
	gun 5 -104 "Javelin Pod"
	turret 0 27.5 "Blaster Turret"
	leak "leak" 60 60
	leak "flame" 40 80
	leak "big leak" 40 20
	explode "tiny explosion" 10
	explode "small explosion" 24
	explode "medium explosion" 28
	explode "large explosion" 10
	"final explode" "final explosion medium"

> 	description "During the caravan era, a boom of economic development spread along the paths of more heavily armed and technologically advanced merchant fleets originating from the Deep, which resulted in ships such as the Saber being developed. This medium warship was one of the powerhouses of military might in its time, armed to the teeth with five gun ports in the nose, and a dorsal turret mount to fire at anything not in front of it. As pirates began to join together in larger war bands in response to the more heavily defended merchants, the Saber became popular amongst them as well. Unlike other ships such as the Firebird, it slowly faded out as competition rose, but it is nevertheless cherished by those who still own one."

Reviewer 4: ```suggestion
	description "During the caravan era, a boom of economic development spread along the paths of more heavily armed and technologically advanced merchant fleets originating from the Deep, which resulted in ships such as the Saber being developed. This medium warship was one of the powerhouses of military might in its time, armed to the teeth with five gun ports in the nose and a dorsal turret mount to fire at anything not in front of it. As pirates began to join together in larger war bands in response to the more heavily defended merchants, the Saber became popular amongst them as well. Unlike other ships such as the Firebird, it slowly faded out as competition rose, but it is nevertheless cherished by those who still own one."
```
